supabase/migrations/20260807182308_canceled_org_app_retention_lifecycle.sql

```
@@ -0,0 +1,362 @@
+COMMENT ON COLUMN public.old_apps.app_created_at IS
+  'Original apps.created_at at archival time.';
+
+CREATE INDEX IF NOT EXISTS old_apps_email_idx
+  ON public.old_apps (email);
+
+CREATE INDEX IF NOT EXISTS old_apps_deleted_at_idx
+  ON public.old_apps (deleted_at);
+
+ALTER TABLE public.old_apps ENABLE ROW LEVEL SECURITY;
+
+DROP POLICY IF EXISTS "Deny all access on old_apps" ON public.old_apps;
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)